test/integration-tests/Seaport.test.ts

```diff
@@ -141,4 +157,187 @@ describe('Seaport', () => {
       router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value: seaportValue })
     ).to.be.revertedWith('ExecutionFailed(0, "0x8baa579f")')
   })
+
+  describe('Seaport SELL 721', async () => {
+    let tubbyCats: ERC721
+    let weth: ERC20
+
+    beforeEach(async () => {
+      await resetFork(16385143) // txn is block 16385144
+      await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [ALICE_ADDRESS],
+      })
+      alice = await ethers.getSigner(ALICE_ADDRESS)
+      permit2 = (await deployPermit2()).connect(alice) as Permit2
+      router = (await deployUniversalRouter(permit2)).connect(alice) as UniversalRouter
+      planner = new RoutePlanner()
+      tubbyCats = new ethers.Contract(TUBBY_ADDRESS, ERC721_ABI) as ERC721
```

Comment:
if tubbycats has to be connected with alice, can do it here so it's one and done instead of having to keep connecting her in each test.

Reply:
Going to fix after refactor to covens